pages/osx/fsck.md

```diff
@@ -0,0 +1,19 @@
+# fsck
+
+> Check the integrity of a filesystem or repair it. The filesystem should be unmounted at the time the command is run. fsck is now essentially a wrapper that fsck_hfs, fsck_apfs, fsck_msdos, fsck_exfat, and fsck_udf as needed.
```

Comment:
Is the 3rd sentence absolutely essential ? The heading is a bit too long IMO for a tldr page.